src/status_im/ui/screens/communities/communities_list_redesign.cljs

```diff
@@ -28,9 +28,10 @@
 (def view-type   (reagent/atom  :card-view))
 (def sort-list-by (reagent/atom :name))
 
-(def mock-community-item-data ;; TO Do remove once communities are loaded with this data.
-  {:data {:status          :gated
+(def mock-community-item-data ;; TODO: remove once communities are loaded with this data.
+  {:data {:status         :gated
           :locked         true
+          :images {:thumbnail {:uri (resources/get-image :status-logo)}}
```

Comment:
If you have Calva installed on VSCode, You can press `CMD + OPTION + L` for Mac or `CTRL + ALT + L` for Linux to align maps.

Reply:
awesome, gonna set this up